profiles/common/modules/features/contact_form/contact_form.install

```diff
@@ -35,4 +35,42 @@ function contact_form_disable() {
   menu_link_delete(NULL, 'contact');
   // Activation message.
   drupal_set_message(t('Contact form feature is now disabled on your site.'));
+  module_disable(array('contact'), FALSE);
```

Comment:
The issue was created by INFO site, so they focus on their problem. They will have to run a hook update to remove the module when moving to 2.3, Tamas is already informed.
We want to have a broader scope. and provide a 'on demand' service that enables contact form module through a feature.
We gradually move all bundled features to 'on demand' features (changed tracked, poetry and now contact)
If you disable poetry feature, poetry module is disabled. This is the same principle except that 'contact' is contrib.
If users want to use contact and not contact_form, they need to create their own feature with contact dependency.